tests/unit/lms/services/auto_grading_test.py

```diff
@@ -115,13 +115,17 @@ def test_get_last_grades(self, svc, db_session, assignment):
             ("all_or_nothing", "cumulative", 15, None, 10, 6, 1),
             ("all_or_nothing", "separate", 10, 5, 10, 4, 0),
             ("all_or_nothing", "separate", 10, 5, 10, 5, 1),
+            ("all_or_nothing", "separate", 10, 0, 9, 4, 0),
```

Comment:
In both of these the 0 value test behaves like in `main`. The actual change is just to handle None.